pawn/tf2items_manager.sp

```diff
@@ -41,8 +37,7 @@ public Plugin:myinfo =
  *
  * When the plugin starts up.
  * -------------------------------------------------------------------------- */
-public OnPluginStart()
-{
+public void OnPluginStart(){

```

Comment:
Prefer spaces not to be crunched - leave a space between the args and the opening brace.